### ISSUE TITLE
[FEATURE SUGGESTION] Ubuntu PPA

### ISSUE THREAD

**Issue by rijnhard** [Soundnode/soundnode-app#780]
Hi 

Instead of downloading the application again for each release it could benefit a lot of users to have a ppa available, that way updates are practically automatic for users.

**Comment by JonasGroeger** [comment]
Duplicate of #775. I'm working on it. Once #777 is merged into `master`, I will provide a package repository.

**Comment by rijnhard** [comment]
@JonasGroeger my apologies, i did try to search, somehow missed it.

**Comment by JonasGroeger** [comment]
Nothing to apologize for :)

**Comment by JonasGroeger** [comment]
@rijnhard Look at https://github.com/Soundnode/soundnode-app/issues/775#issuecomment-224342781

**Comment by rijnhard** [comment]
I'll give it a go and get back to you. (Going to set it up in a docker
container)
